### ISSUE TITLE
Catalepsy

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11037]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
A trancelike state marked by loss of voluntary motion in which the limbs remain in whatever position they are placed (https://www.merriam-webster.com/dictionary/echo#medicalDictionary)

**Parent term (use hpo.jax.org/app)**
Abnormality of movement HP:0100022

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Idiopathic catatonia ORPHA:648919

**Your nano-attribution (ORCID)**
0009-0005-6714-5727
